test/cli.js

```diff
@@ -180,4 +195,59 @@ test('cli', async (t) => {
     const {stdout} = await execa(cli, args, opts)
     t.equal(stdout.trim(), logs.trim(), 'input piped to stdout')
   })
+
+  t.test('--empty-message changes default message', async (t) => {
+    t.plan(4)
+
+    var proc
```

Comment:
q: is there a significance for the `var` keyword here?

Reply:
The linter was complaining about `let proc` never being reassigned and thus wanted me to convert it to a `const`. That... would not work. So I made the linter happy with a `var`.

Reply:
fair nuff